digital-deck-server/index.js

```diff
@@ -14,9 +13,9 @@ app.use(bodyParser.json());
 app.use('/session', sessionRoute);
 app.use('/player', playerRoute);
 
-
-const server = app.listen(port);
```

Comment:
Had to remove this to run multiple test suites. Not sure if this was used for anything. Doesn't seem like there's any error for removing it

Reply:
This was required for mongodb. we will figure out some way later to run them both after this sprint